docs/learn-faq.md

```diff
@@ -6,80 +6,327 @@ sidebar_label: Frequently Asked Questions (FAQs)
+voting, and imbues greater voting power to token holders who are willing to lock
+their tokens on the protocol for longer lengths of time. Longer lock-ups display _conviction_
+in a vote. Conviction voting could allow a highly determined minority to overrule
+the vote of an apathetic majority in certain situations. Another mechanism is
+known as Adaptive Quorum Biasing, and this makes it so that proposals have a
+varying threshold for approval or rejection based on what part of the governance
+protocol that the proposal originated in. For details on the subtleties of
+Polkadot's governance system, please see the [governance page](learn-governance).
+
+## Parachains
+
+### How do parachain economics work?
+
+Parachains have the flexibility to implement their own monetary system or
+incentive structure for collators. However, this is not strictly necessary.
+Since the collator's job is to continue to give recent state transitions to
+the validators on the relay chain whom validate each transition, the security
+of the parachain and the Polkadot network is completely separate from parachain
+economics. Parachains need collators to continue to progress, so it wouldn't be
+unreasonable to see them incentivize collator nodes in some way but it is completely
+up to parachain implementers.
+
+### Are parachains ephemeral? What happens when a parachain loses the next auction?
+
+Parachains are not ephemeral. As long as someone is keeping the data for a
+parachain, the parachain can move between being a parachain, a parathread, or
+a separate sovereign chain at different points of its lifetime. Especially
+with parathreads, parachains can be decommissioned to only produce blocks when
+their usage and throughput makes it necessary.
+
+When a parachain loses an auction for renewal, that parachain has a few options.
+In most cases, becoming a parathread instead would be a suitable choice. Parathreads
+are still secured by the Relay Chain, but don't need to hold a parachain slot
+and can produce a block when its economically feasible for them. For more on
+parachains please see the [parachains page](learn-parachains) and for more on
+parathreads see [here](learn-parathreads).
+
+## Networking
+
+### What is libp2p?
+
+[Libp2p][libp2p] is a modular and extensible networking stack that is used by
+IPFS, Substrate, and many other projects. It is a collection of peer-to-peer
+protocols for finding peers and connecting to them. It's modules have logic for
+content routing, peer routing, peer discovery, different transports, and NAT
+traversals. It is intended to be used by applications for building large scale
+peer-to-peer networks by only selecting the parts of the protocol suite that are
+needed.
+
+The Rust implementation of the specification was built and primarily maintained
+by a team of contributors at Parity Technologies. The Go and JavaScript versions
+are maintained by Protocol Labs as well as community contributors. It is an open
```

Comment:
There is now also a nim-native version used by Nimbus, if that warrants a mention.